doc/source/models/source/source.rst

```diff
@@ -0,0 +1,54 @@
+.. _models_download:
+
+===============
+Download Source
```

Comment:
Given the presence of multiple sources, renaming it to Download Sources is more appropriate.

Please remember to update the filename and the index accordingly.